src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -1849,3 +2162,142 @@ def match_attachment(request):
             document_create_attachment_body["vertrouwelijkheidaanduiding"], "geheim"
         )
         self.assertEqual(document_create_attachment_body["titel"], "A Custom Title")
+
+    @override_settings(OBJECTS_API_DATA_SIZE_LIMIT=1000000)
+    def test_submission_with_objects_api_content_json_exceed_max_file_limit(self, m):
+        # generates template of 1000007 bytes
+        self.config.content_json = {"oversized_list": list(range(0, 138875))}
+        self.config.save()
+        self.config.refresh_from_db()
```

Comment:
this is making your data big and therefore the test slow

It's far easier to just use a stupid small file size limit: `@override_settings(OBJECTS_API_DATA_SIZE_LIMIT=10)` -> 10 bytes, that'll trip everything.

You also really want to avoid making actually database/cache calls to singleton models, for those `unittest.mock.patch` is appropriate. There are plenty of other tests where this happens, so please apply the same pattern.